### ISSUE TITLE
should_get_only_private_analysis causes HTTP server error

### ISSUE THREAD

**Issue by cccs-kevin** [intezer/analyze-python-sdk#35]
Hi there!

I'm trying to get the latest analysis via the `get_latest_analysis` method for a SHA256 from my private Intezer Analyze instance, and I'm getting the following error:

```
Error Stacktrace:
    res = self.client.get_latest_analysis(file_hash=sha256)
  File "/path/lib/python3.9/site-packages/intezer_sdk/api.py", line 165, in get_latest_analysis
    raise_for_status(response)
  File "/path/lib/python3.9/site-packages/intezer_sdk/api.py", line 54, in raise_for_status
    raise requests.HTTPError(http_error_msg, response=response)
HTTPError: 400 Client Error: BAD REQUEST for url: http://<private Intezer Analyze instance IP>/api/v2-0/files/<sha256>, server returns The request is not valid, details: {'should_get_only_private_analysis': ['unknown field']}
```

I can use the REST endpoint easily, but when I use the client this parameter passed as an option seems to cause problems https://github.com/intezer/analyze-python-sdk/blob/d4fad0d895b993f16b1f9f0ecdea36c577673c51/intezer_sdk/api.py#L157

**Comment by davidt99** [comment]
Hi @cccs-kevin,
There is a difference between the cloud API and the on-premise version, mostly the cloud API advancing faster than the on-premise till we deliver a new on-premise version.
I'll soon address this in the SDK, I'll mention you with the details regarding how to configure the SDK to work with the on-premise correctly.

**Comment by cccs-kevin** [comment]
> @cccs-kevin this was fixed in 1.8.0. Please specify your on-premise version when initializing the API:
> 
> ```python
> from intezer_sdk.consts import OnPremiseVersion
> from intezer_sdk.api import IntezerApi
> 
> api = IntezerApi(on_premise_version=OnPremiseVersion.V21_11)
> ```

Fantastic, thank you so much!
